internetarchive/utils.py

```diff
@@ -234,15 +244,19 @@ def recursive_file_count(files, item=None, checksum=False):
             except (AttributeError, TypeError):
                 is_dir = False
         if is_dir:
-            for x, _ in iter_directory(f):
-                lmd5 = get_md5(open(x, 'rb'))
+            pb = tqdm(iter_directory(f), ascii=True,
+                      desc='Checking md5 for duplicate files')
+            for x, _ in pb:
+                with open(x, 'rb') as f_handle:
+                    lmd5 = get_md5(f_handle)
```

Comment:
is it really worth doing a progress bar on md5 checks? how often is that slow enough to warrant such a change?

and doesn't it overlap with the previous progress bar?